src/fastoad/model_base/atmosphere.py

```diff
@@ -182,6 +183,23 @@ def true_airspeed(self) -> Union[float, Sequence[float]]:
                 )
             if self._unitary_reynolds is not None:
                 self._true_airspeed = self._unitary_reynolds * self.kinematic_viscosity
+            if self._calibrated_airspeed is not None:
+                sea_level = Atmosphere(0)
+                current_level = Atmosphere(self._altitude, altitude_in_feet=False)
+                impact_pressure = sea_level.pressure * (
+                    (
+                        (np.asarray(self._calibrated_airspeed) / sea_level.speed_of_sound) ** 2.0
+                        / 5.0
+                        + 1.0
+                    )
+                    ** 3.5
+                    - 1.0
+                )
+                total_pressure = current_level.pressure + impact_pressure
+                sigma_0 = total_pressure / current_level.pressure
+                gamma = 1.4
+                mach = (2.0 / (gamma - 1.0) * (sigma_0 ** ((gamma - 1.0) / gamma) - 1.0)) ** 0.5
```

Comment:
Here your formula explicitly uses gamma, but in the formula above, you use `3.5`, which is &#947;/(&#947;-1). For sake of consistency, the `gamma` variable should be used everywhere, or nowhere. And for sake of efficiency, I think that nowhere is the best choice.

Therefore, I suggest:
```suggestion
                mach = (5.0 * (sigma_0 ** (1.0 / 3.5) - 1.0)) ** 0.5
```